src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -618,6 +617,10 @@ public async Task Initialize(HostWaitLoop waitLoop, CancellationToken token)
                         }
                     }
                 }
+                // now the exe is loaded and we can check target features
+                TargetFeatures = await MICommandFactory.GetTargetFeatures();
+                if (TargetFeatures.Contains("async"))
+                    await MICommandFactory.EnableTargetAsyncOption();
```

Comment:
To support the lldb-mi case, if GetTargetFeatures fails, we need to always run `EnableTargetAsync`